<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="submitRating">
   <property name="geometry">
    <rect>
     <x>444</x>
     <y>370</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit Rating</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>111</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="MyRes.qrc">:/new/prefix1/default_large.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Waiter ID:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Restaurant</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="orderedItems">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>511</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>New Column</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Rate*</string>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>391</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="waiterRate">
      <property name="text">
       <string>Good</string>
      </property>
      <property name="autoRepeat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="waiterRate_2">
      <property name="text">
       <string>Average</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton">
      <property name="text">
       <string>Bad</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>id</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>481</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>*Choose -1 for Bad, 0 for Average, 1 for Good</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="MyRes.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>submitRating</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>onSubmitRatingClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>489</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>setWaiterRating()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>476</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>waiterRate</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>setWaiterRating()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>waiterRate_2</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>setWaiterRating()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>345</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onSubmitRatingClicked()</slot>
  <slot>setWaiterRating()</slot>
 </slots>
</ui>
